src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -826,7 +821,7 @@ private void appendAppUIDToAccount(StorageHelper cryptoHelper, Account account)
             } else {
                 try {
                     appIdList = cryptoHelper.decrypt(appIdList);
-                } catch (Exception ex) {
+                } catch (GeneralSecurityException | IOException ex) {
```

Comment:
I think we should log the specific exception out of GeneralSecurityException. @weijjia